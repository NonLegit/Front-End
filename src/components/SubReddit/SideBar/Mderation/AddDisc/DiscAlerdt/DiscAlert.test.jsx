import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import DistAlert from './DistAlert';

// test snapshot
test('test snapshot', async () => {
  const tree = renderer.create(<DistAlert />).toJSON();
  expect(tree).toMatchSnapshot();
});
