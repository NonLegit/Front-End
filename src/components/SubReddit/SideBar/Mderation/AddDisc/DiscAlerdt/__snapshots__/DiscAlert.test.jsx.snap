// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot 1`] = `
<div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1f5eo08-MuiButtonBase-root-MuiButton-root"
    disabled={false}
    id="DiscAlert"
    onBlur={[Function]}
    onClick={[Function]}
    onContextMenu={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    openForm dialog
  </button>
</div>
`;
